Content.Shared/Cargo/SharedPriceGunSystem.cs

```diff
@@ -40,6 +42,7 @@ private void OnAfterInteract(Entity<PriceGunComponent> entity, ref AfterInteract
         if (!args.CanReach || args.Target == null || args.Handled)
             return;
 
+        _audio.PlayPredicted(entity.Comp.AppraisalSound, entity.Owner, args.User);
```

Comment:
This bypasses the use delay, meaning you can spam the sound by clicking repeatedly.
Also the sound does not play if you use the verb in the right click menu.
I would move playing the sound to the server into the `GetPriceOrBounty` function to fix this, which will also better sync up the sound with the popup since it is not predicted. You may have to use `_audio.PlayPvs` in that case.